Appserver/Handlers/TextractHandler.cs

```diff
@@ -51,6 +82,138 @@ private async Task<AnalyzeDocumentResponse> StartDocumentAnalysis(Stream file, L
 			request.FeatureTypes = featureTypes;
 			var response = await this.textractClient.AnalyzeDocumentAsync(request);
 			return response;
-		}
+		}
+
+		// In order for textract to process PDFs, they must come from an S3 bucket
+		// that is in the same region that textract is in. The file is no longer
+		// needed in S3 after being processed by textract as the file is still
+		// being stored in azure blob.
+        // TODO make this work? Seems less complicated if it can.....
+		private async Task<AnalyzeDocumentResponse> StartDocumentPDFAnalysis(IFormFile file, List<string> featureTypes)
+		{
+			// Upload PDF to S3, with guid as file key
+			var k = Guid.NewGuid();
+			PDFtoS3Bucket(file, k.ToString()).Wait();
+
+
+			var s3 = new Amazon.Textract.Model.S3Object();
+            s3.Bucket = "multcapstone";
```

Comment:
The name of the bucket should not be hard coded in here. Look at BLOB_CONNECTION for a way to handle this similarily.

Reply:
See latest commit.